.github/workflows/merge.yml

```
@@ -213,6 +215,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v6
 
+      - name: Mask inherited secrets
+        run: echo "::add-mask::${{ needs.terraform.outputs.upstash_redis_url }}"
```

actionlint (1.7.12)

[error] property "upstash_redis_url" is not defined in object type {vm_host: string}

(expression)